<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="pch-ee210988" failures="1" tests="8" name="Test" time="29.388" errors="0" timestamp="2020-02-04T14:30:12 IST">
  <testcase classname="com.test.parabank.TestHomePage" name="testCustomerCare" time="12.942"/>
  <testcase classname="com.test.parabank.TestAdminPage" name="adminTest" time="5.634"/>
  <testcase classname="com.test.parabank.TestAdminPage" name="failTheTest" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [-1] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [-1] but found [1]
at com.test.parabank.TestAdminPage.failTheTest(TestAdminPage.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- failTheTest -->
  <testcase classname="com.test.parabank.TestAdminPage" name="triggerBrowser" time="0.816"/>
  <testcase classname="com.test.parabank.TestAdminPage" name="applicationSettings" time="0.045"/>
  <testcase classname="com.test.parabank.TestAdminPage" name="testAccessMode" time="0.928"/>
  <testcase classname="com.test.parabank.TestAdminPage" name="testApplicationSettings" time="7.185"/>
  <testcase classname="com.test.parabank.TestAdminPage" name="testWebService" time="1.635"/>
  <testcase classname="com.test.parabank.TestHomePage" name="testAboutUs" time="0.0">
    <ignored/>
  </testcase> <!-- testAboutUs -->
</testsuite> <!-- Test -->
